internal/resolution/resolve.go

```diff
@@ -104,6 +114,92 @@ func Resolve(ctx context.Context, cl client.ResolutionClient, m manifest.Manifes
 	return result, nil
 }
 
+func resolvePostProcess(ctx context.Context, cl client.ResolutionClient, m manifest.Manifest, opts ResolveOpts, graph *resolve.Graph) (*resolve.Graph, error) {
+	if m.System() == resolve.Maven && opts.MavenManagement {
```

Comment:
I prefer returning first if it does not require further processing (just my preference).

Reply:
I was thinking in the future there may be some other post-processing steps (e.g. possibly dealing with npm `peerDependencies` here), so we wouldn't want to immediately return in that case.